### PR TITLE
[LMS-340][FE][Markup] Home Page

### PR DESCRIPTION
**Pull request by FranzDelosSantos**
## Issue Link
https://framgiaph.backlog.com/board/LMS?selectedIssueKey=LMS-340&assignee=417519

## Defintion of Done
Created /home  for the homepage
Logout button should be visible in the navbar
Random Content should be displayed
Navbar should be sticky and must remain in the top always upon scrolling up and down 

## Steps to reproduce
docker compose up -d
visit url: http://localhost:3001/home

## Affected Components / Functionalities / Page
navbar component
pages/home/index.tsx

## Notes
n/a

## Screenshots
image

### CODE REVIEW

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
- [ ] Created /home for the homepage.
- [ ] Logout button is visible in the navbar.
- [x] Random Content is displayed.
- [x] Navbar is sticky and remained in the top upon scrolling up and down.
- [x] No ESLint warnings or errors.


Comments: 0
Test time: 31 mins
Iterations: 0

Proof:
home

File: client/src/pages/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      { text: 'Courses', url: '/catalog/courses' },
+      { text: 'Learning Paths', url: '/catalog/learning-paths' }
+    ]
+  }
+];
+export const dropdownItems = [
+  { text: 'Profile', url: '/profile' },
+  { text: 'Settings', url: '/settings' },
+  { text: 'Logout', url: '/logout' }
+];
```
Comment (kdacudag) [inline client/src/pages/home/index.tsx:19]: Please use the nav items and drop down items [here](https://drive.google.com/drive/folders/11QiaCRhzxcSUwZMuRbwHlEe5SUxY8eht?usp=share_link)

It's okay if the dropdown items' link are not yet created.

You can move these items to another component.

File: client/src/pages/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Navbar from '@/src/shared/components/Navbar';
+import { dropdownItems, navItems } from '@/src/shared/utils/navBarList';
+import { Fragment, type ReactNode } from 'react';
```
Comment (kdacudag) [inline client/src/pages/home/index.tsx:3]: Move to top-most since this is an external package

Comment (kdacudag) [inline client/src/pages/home/index.tsx:3]: Move react to top-most

**Review by kdacudag — APPROVED** [review]
Approved
Comments: 3
Iteration: 2

**Pull request merged**